<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notice</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/src.qrc">
    <normaloff>:/images/a16px.png</normaloff>:/images/a16px.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="dialogWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#dialogWidget{
background-color:#222222;
border-radius:6px;
border:0.5px solid #464646;
}</string>
   </property>
   <widget class="QWidget" name="bodyWidget" native="true">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>44</y>
      <width>312</width>
      <height>152</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bodyWidget{
background-color:#2d2d2d;
border:none;
}</string>
    </property>
    <widget class="QPushButton" name="accept">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>87</y>
       <width>114</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#accept{
border:none;
color:#fff;
background-color:#2eb3ff;
border-radius:6px;
}
QPushButton#accept:hover{
border:none;
color:#fff;
background-color:#3fc4ff;
border-radius:6px;
}
QPushButton#accept:pressed{
border:none;
color:#fff;
background-color:#1da2ff;
border-radius:6px;
}</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel">
     <property name="geometry">
      <rect>
       <x>174</x>
       <y>87</y>
       <width>114</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#cancel{
border:none;
color:#fff;
background-color:#393939;
border-radius:6px;
}

QPushButton#cancel:hover{
background-color:#464646;
border:none;
color:#fff;
border-radius:6px;
}

QPushButton#cancel:pressed{
background-color:#343434;
border:none;
color:#fff;
border-radius:6px;
}</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>24</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{
color:#fff;
}</string>
     </property>
     <property name="text">
      <string>Confirm to exit Vaudit Audit?</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="titleBar" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>318</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#titleBar{
background-color:#2d2d2d;
border-top-left-radius:6px;
border-top-right-radius:6px;
}</string>
    </property>
    <widget class="QLabel" name="icon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/a16px.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="titleText">
     <property name="geometry">
      <rect>
       <x>37</x>
       <y>3</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#fff;</string>
     </property>
     <property name="text">
      <string>Notice</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../data/src.qrc"/>
 </resources>
 <connections/>
</ui>
